src_assets/common/assets/web/config.html

```diff
@@ -517,6 +517,21 @@ <h1 class="my-4">Configuration</h1>
           <pre>   0: +HDMI-1 1920/518x1200/324+0+0 HDMI-1</pre>
         </div>
       </div>
+      <div class="mb-3">
+          <label for="qp" class="form-label">Preloaded Frame Count</label>
+          <input
+            type="number"
+            class="form-control"
+            id="min_fps_target"
+            placeholder="3"
```

Comment:
Placeholder value should be the default.

```suggestion
            placeholder="12"
```